core/src/View/Kokkos_ViewCtor.hpp

```diff
@@ -213,11 +211,12 @@ struct ViewCtorProp : public ViewCtorProp<void, P>... {
   using execution_space = typename var_execution_space::type;
   using pointer_type    = typename var_pointer::type;
 
-  /*  Copy from a matching argument list.
-   *  Requires  std::is_same< P , ViewCtorProp< void , Args >::value ...
-   */
```

Comment:
Why did you remove this comment?

Reply:
I removed the comment because now that the function passes by universal reference, I thought the comment might become less readable, because we'd have to remove references and const. All in all, it looked like by keeping the line "requires a matching argument list", things would remain sufficiently clear. (just a detail, it seems the code snippet is also missing the type alias `type` from `ViewCtorProp<void , Args>`)

Reply:
We added a modified version of this constraint directly in the constructor in the code.